cmd/tendermint/commands/init.go

```diff
@@ -18,28 +16,31 @@ var InitFilesCmd = &cobra.Command{
 
 func initFiles(cmd *cobra.Command, args []string) {
 	privValFile := config.PrivValidatorFile()
-	if _, err := os.Stat(privValFile); os.IsNotExist(err) {
-		privValidator := types.GenPrivValidatorFS(privValFile)
+	var privValidator *types.PrivValidatorFS
+	if cmn.FileExists(privValFile) {
+		privValidator = types.LoadPrivValidatorFS(privValFile)
+		logger.Info("Already initialized", "priv_validator", privValFile)
+	} else {
+		privValidator = types.GenPrivValidatorFS(privValFile)
 		privValidator.Save()
+		logger.Info("Initialized tendermint", "privValidator", privValFile)
```

Comment:
let's change messages to something more clear, like `Initialized private validator` and `Found private validator`